apps/api-extractor/src/api/ExtractorConfig.ts

```diff
@@ -41,9 +42,35 @@ interface IExtractorConfigTokenContext {
    */
   packageName: string;
 
+  /**
+   * The `<projectFolder>` token returns the expanded `"projectFolder"` setting from api-extractor.json.
+   */
   projectFolder: string;
 }
 
+/**
+ * Options for {@link ExtractorConfig.tryLoadForFolder}.
+ *
+ * @public
+ */
+export interface IExtractorConfigLoadForFolderOptions {
+  /**
+   * The folder path to start from when searching for api-extractor.json.
+   */
+  startingFolder: string;
```

Comment:
Maybe `projectFolder`?

Reply:
Somehow this got super complicated heheh. The `"projectFolder"` is a setting that is specified in **api-extractor.json** itself.  It may be different from the `packageFolder` that contains **package.json**, and there are valid configurations where there is no **package.json** file at all.  Whereas `startingFolder` is usually just `process.cwd()` which might be a subfolder.